rector.php

```diff
@@ -52,6 +53,7 @@
 return static function (ContainerConfigurator $containerConfigurator): void {
     $containerConfigurator->import(SetList::DEAD_CODE);
     $containerConfigurator->import(SetList::PHP_73);
+    $containerConfigurator->import(SetList::PHP_74);
```

Comment:
Do we still need to include SetList::PHP_73 here?

Reply:
Yes, that means php 7.3 syntax will be included as well.

Reply:
See example of multiple php setlist so all features from php-x to php-y applied https://github.com/samsonasik/ErrorHeroModule/blob/master/rector.php#L21-L26